src/cr/cube/matrix/assembler.py

```diff
@@ -254,6 +252,15 @@ def rows_margin(self):
             self._cube_result_matrix.rows_margin, self._row_subtotals, self._row_order
         )
 
+    @lazyproperty
+    def sum(self):
+        """2D optional np.float64 ndarray of sum for each cell.
+        Raises `ValueError` if the cube-result does not include a sum cube-measure.
+        """
+        return self._assemble_matrix(
+            NanSubtotals.blocks(self._cube_result_matrix.sum, self._dimensions)
+        )
```

Comment:
Nothing new should be using `._cube_result_matrix` because that's the legacy way. This should be something like:

`self._assemble_matrix(self._measures.sum.blocks)`

See how `Assembler.weighted_counts` works and model your approach after that. Basically:

1. `Assembler calls `._assemble_matrix` on a collection of blocks (2x2 list of lists).
2. blocks are provided by the appropriate measure, so probably there is a new `_Sum` measure added to `cr.cube.matrix.measures` and that provides the blocks. The blocks are body values, subtotal-rows, subtotal-columns, and intersections.
3. The measure computes the blocks with the help of a cube-measure (like maybe a new `_CubeSums` object, possibly a `_BaseCubeSums` object and `CatXCat, CatXMr, MrXCat, and MrXMr` subclasses, depending on the variety required by other dimensions. The measure also computes the subtotatals, often with the help of a Subtotals collaborator like perhaps `NanSubtotals` in this case.

Reply:
I think i did it now, included the `Mean`  measure